modules/kafka/src/main/scala/com.snowplowanalytics.snowplow.enrich.kafka/Main.scala

```diff
@@ -60,7 +61,7 @@ object Main extends IOApp.WithContext {
       List.empty,
       _.record.value,
       MaxRecordSize,
-      None,
+      Some(Cloud.Azure),
```

Comment:
This could be a bit misleading... since someone can run Enrich Kafka outside of Azure

Reply:
Yeah I thought about that. But if we don't do that we need to make the cloud configurable. Given that this is only for telemetry for now, and that we are not even sure to keep it (unless I missed something), I assumed that this was the best call. Do you think that I should change that?

Reply:
Is there anything in the Kafka config for Azure that we could use to identify that we are running there? I know we need to set these properties, but they seem a bit generic...

```
// Event Hubs specific settings using a connection string
props.setProperty(“security.protocol”, “SASL_SSL”)
props.setProperty(“sasl.mechanism”, “PLAIN”)
props.setProperty(“sasl.jaas.config”, “”“org.apache.kafka.common.security.plain.PlainLoginModule required username=”$ConnectionString" password=“<your_connection_string>”;“”")

// Event Hubs specific settings for OAuth
props.setProperty(“security.protocol”, “SASL_SSL”)
props.setProperty(“sasl.mechanism”, “OAUTHBEARER”)
props.setProperty(“sasl.jaas.config”, “org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required;”)
props.setProperty(“sasl.login.callback.handler.class”, “CustomAuthenticateCallbackHandler”)
```

Reply:
I'm afraid that there isn't anything we can use, as these security parameters are generic and can be used outside of Azure

Reply:
I reverted the change. We should address this when we add support for `BlobStorage` to `enrich-kafka`